kerngen/tests/test_kerngen.py

```diff
@@ -1,10 +1,15 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
 # Copyright (C) 2024 Intel Corporation
```

Comment:
You will hit this problem as the pre-commit that does this may need config. I think @faberga dealt with something similar before.
```suggestion
# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```